app/controllers/support/user_permissions_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Support
+  class UserPermissionsController < SupportController
+    def destroy
+      if user.remove_access_to(provider)
+        redirect_to origin_page, flash: { success: "#{user_full_name} removed from #{provider.provider_name}" }
```

Comment:
We've got some generic flash messages for CRUD actions, see the `app/controllers/support/locations_controller.rb` for eg. Would be good to maybe keep this consistent